src/services/sendEmail.js

```diff
@@ -0,0 +1,18 @@
+// using Twilio SendGrid's v3 Node.js Library
+// https://github.com/sendgrid/sendgrid-nodejs
+const sgMail = require('@sendgrid/mail');
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const sendEmail = msg => {
```

Comment:
This should return a `Promise`

```
const sendEmail = async msg => {
  return sgMail
    .send(msg)
    .then(() => {
      console.log(`Email sent to ${msg.to}`);
    })
    .catch(error => {
      console.error(error);
      throw(error)
    });
};
```

Also note that we `throw` the error after catching it - imagine that `console.log` was actually a system logger. The service does some logging, but throws the error again down the promise chain.